packages/components/src/composite/current/stories/index.story.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Composite, CompositeRow, CompositeItem, useCompositeStore } from '..';
+
+type InitialState = Parameters< typeof useCompositeStore >[ 0 ];
+
+const Placeholder = ( _: InitialState ) => <></>;
+
+const meta: Meta< typeof Placeholder > = {
+	title: 'Components/Composite/Composite (New)',
+	id: 'components-composite-new',
+	component: Placeholder,
+};
+export default meta;
```

Comment:
Noting that this story doesn't have any props / controls displayed in Storyook

![Screenshot 2024-01-17 at 17 13 19](https://github.com/WordPress/gutenberg/assets/1083581/5bdc51f4-8b09-497d-b96c-5dd3a151744f)

Reply:
I "fixed" this, per [another conversation](https://github.com/WordPress/gutenberg/pull/57304#discussion_r1456438029)...

![Legacy composite docs, showing tabs for 'useCompositeStore', 'Composite', 'CompositeGroup', 'CompositeRow' and 'CompositeItem'](https://github.com/WordPress/gutenberg/assets/159848/d916b33f-8321-422c-ae49-9de5078632a3)